Source/pack.cpp

```diff
@@ -80,31 +80,24 @@ void __fastcall PackPlayer(PkPlayerStruct *pPack, int pnum, BOOL manashield)
 
 void __fastcall PackItem(PkItemStruct *id, ItemStruct *is)
 {
-	short v2; // ax
-	short v3; // bx
-
 	if (is->_itype == -1) {
-		id->idx = -1;
+		id->idx = 0xFFFF;

```

Comment:
Maybe worth assigning this via cast?  Like:
`id->idx = (WORD)-1;`

Reply:
Interesting, however we haven't done so any where else.

Reply:
Maybe just during earlier cleanup iterations no one made such thought. 

OTOH, this could be done earlier if needed. It just looks more human readable to me.